### ISSUE TITLE
xor + 목적 함수

### ISSUE THREAD

**Issue by yuseungwoo** [yuseungwoo/pytorch#5]
import torch 
import torch.nn as nn

torch.manual_seed(70)   

class Network2(nn.Module):
    def __init__(self,i_size,h_size,o_size):
        super(Network2,self).__init__()
        self.ih=nn.Linear(i_size,h_size)
        self.ho=nn.Linear(h_size,o_size)
        self.sig=nn.Sigmoid()
    
    def forward(self,x):
        z1=self.ih(x)
        a1=self.sig(z1)
        z2=self.ho(a1)
        a2=self.sig(z2)
        
        return a2

x=torch.Tensor([[0,1]])
t = torch.Tensor([1])
net = Network2(2,2,1)
y=net(x)

loss_fuction = nn.KLDivLoss(reduction="sum")
e=1e-30

one_hot=torch.eye(2)
prob_t=one_hot.index_select(dim=0, index=t.long())
prob_y=torch.cat([1-y,y], dim=1)

print(one_hot)
print(prob_t)
print(prob_y)
print(loss_fuction(torch.log(prob_y+e),prob_t+e))
